.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+name: ci
+
+on: [push, pull_request]
```

Триггер нужен только на pull request

---

аргументация будет?
и да, ты можешь делать изменения в ветку пра
```suggestion
on: [pull_request]
```